web/src/components/PreflightResultPage.jsx

```diff
@@ -312,9 +312,10 @@ class PreflightResultPage extends Component {
     const { slug } = this.props.match.params;
     const { showSkipModal, showWarningModal, preflightResultData, errorMessage } = this.state;
 
-    const stopPolling = !!preflightResultData?.result;
+    const preflightSkipped = preflightResultData?.skipped;
+    const stopPolling = (preflightResultData?.result || preflightSkipped);
     let preflightJSON = {};
-    if (stopPolling) {
+    if (stopPolling && !preflightSkipped) {
```

Comment:
how about doing this instead?
```suggestion
    if (preflightResultData?.result) {
```

Reply:
hmm, you don't think this is easier to read?